hummingbot/client/command/connect_command.py

```diff
@@ -102,6 +108,33 @@ async def connection_df(self  # type: HummingbotApplication
                         failed_msgs[option] = err_msg
                     else:
                         keys_confirmed = 'Yes'
+                data.append([option, keys_added, keys_confirmed])
+                """
```

Comment:
if you don't need this section, better remove it.

Reply:
Thanks, again. 
I'd scan through the code again to ensure that it's clean.

Reply:
Okay.